Zend/zend_execute.c

```diff
@@ -1504,10 +1504,14 @@ static zend_never_inline ZEND_COLD void ZEND_FASTCALL zend_wrong_property_read(z
 
 static zend_never_inline ZEND_COLD void ZEND_FASTCALL zend_deprecated_function(const zend_function *fbc)
```

Comment:
I'd suggest making this `ZEND_API` and reusing it in the other places that generate the same message. (Would also come in handy if we get custom deprecation messages through attributes...)